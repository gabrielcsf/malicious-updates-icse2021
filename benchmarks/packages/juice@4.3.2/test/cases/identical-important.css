div {
    color: black !important;
}

div {
    color: blue !important;
}