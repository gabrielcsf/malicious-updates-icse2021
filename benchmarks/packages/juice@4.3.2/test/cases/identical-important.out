<div style="color: blue;"></div>